ts/components/RegionServiceWebView.tsx

```diff
@@ -110,79 +112,40 @@ const RegionServiceWebViewPlayGround: React.FunctionComponent<Props> = (
         return;
       case "SHOW_SUCCESS":
         setSuccess(true);
-        setText(data.payload[locale]);
+        setText(data[locale]);
```

Comment:
instead of access every time `data[locale]` you could place it into a const
`const localizedData = data[locale]`

Reply:
the attribute doesn't always exists for all the message types: `CLOSE_MODAL`, `START_LOADING`, `END_LOADING` types don't require a message payload

Anyway i just made an improvement: if the data in current locale is not available (i.e. `it`) will fallback on english one (mandatory)